client/main.lua

```diff
@@ -58,13 +59,26 @@ AddAudioSubmixOutput(phoneEffectId, 1)
 
 local submixFunctions = {
 	['radio'] = function(plySource)
+		MumbleSetVolumeOverrideByServerId(plySource, radioVolume or volume)
```

Comment:
You're going to double set their volume

Reply:
Works fine though, doing this in prod already.